docs/software-platforms/aws/index.md

```diff
@@ -3,24 +3,20 @@
 We use Amazon Web Services (AWS) to grant contributors access to data and Linux virtual computers.
 
 !!! note
-    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about AWS access.
+    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.
 
-## Joining IAM Identity Center
+    Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).
```

Comment:
Since I was assigned to #258, I started to stack on this... but I no longer think that makes sense. Can you add (suggestion is not working for me) the following here:

```
    Your use of AWS services described in this section counts towards your [monthly budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
```

Reply:
> but I no longer think that makes sense. Can you add (suggestion is not working for me) the following here:

Replace or just add?

Reply:
Add